bin/hipfc

```diff
@@ -205,7 +205,7 @@ while [ $# -gt 0 ] ; do
       -lcurand)         CUDA_LIBS+=" curand" ;;
       -triple)          TARGET_TRIPLE=$2; shift ;;
       -cuda-dir)        CUDA_HOME=$2; shift ;;
-      -rocm-dir)        ROCM_DIR=$2; shift ;;
+      -rocm-dir)        ROCM_PATH=$2; shift ;;
```

Comment:
I would prefer if we change the argument to `-rocm-path` to be a little more consistent.
We should probably switch to `-cuda-path` or `-cuda-home` from `-cuda-dir`.

Reply:
We can do as -cuda-path instead of -cuda-dir

Reply:
If I'm understanding correctly, we're going with
`-cuda-path` and `CUDA_PATH`
`-rocm-path` and `ROCM_PATH`

This would be ideal.